Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2977,6 +2977,8 @@ def gen2018HiMix(fragment,howMuch):
     	PUDataSets[ds]={'-n':10,'--pileup':'AVE_35_BX_25ns','--pileup_input':'das:/RelValMinBias_13/%s/GEN-SIM'%(name,)}
     elif '2018' in name:
     	PUDataSets[ds]={'-n':10,'--pileup':'AVE_50_BX_25ns','--pileup_input':'das:/RelValMinBias_13/%s/GEN-SIM'%(name,)}
+    elif '2019' in name:
+    	PUDataSets[ds]={'-n':10,'--pileup':'AVE_50_BX_25ns','--pileup_input':'das:/RelValMinBias_13/%s/GEN-SIM'%(name,)}
```

Comment:
The Run3 GT 105X_postLS2_realistic_v6-v1 does NOT have the string "2019". So these two lines will not work for pileup.

Reply:
Ah too bad this is unfortunate that Alca didn't  follow the usual  pattern, I will make a workaround  meanwhile and add the minbias you produced -  Thanks for having noticed